catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryObjectMetacardType.java

```diff
@@ -0,0 +1,115 @@
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.ORGANIZATION_PHONE_NUMBER)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.ORGANIZATION_EMAIL)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.POINT_OF_CONTACT)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.LIVE_DATE)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.DATA_START_DATE)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.DATA_END_DATE)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.LINKS)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.GEOGRAPHY)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.REGION)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.DATA_SOURCES)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.DATA_TYPES)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.SERVICE_BINDINGS)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.SERVICE_BINDING_TYPES)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.REGISTRY_ID)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.REGISTRY_IDENTITY_NODE)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.REGISTRY_LOCAL_NODE)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.REGISTRY_BASE_URL)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.PUBLISHED_LOCATIONS)
+                .isStored(), is(true));
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(registryObjectMetacardType.LAST_PUBLISHED)
```

Comment:
I guess I just assumed that transient attributes wouldn't be stored? If that's not the case, then what is testable about transient attributes?